package.json

```
@@ -50,22 +50,22 @@
   },
   "dependencies": {
     "accept-language": "^3.0.18",
-    "bcrypt": "^1.0.3",
-    "concolor": "^0.1.7",
+    "bcrypt": "^3.0.4",
+    "concolor": "^0.1.11",
     "csv": "^2.0.0",
     "globalstorage": "0.2.0",
-    "iconv-lite": "^0.4.23",
-    "maojian": "^0.0.5",
-    "metarhia-common": "^0.0.25",
-    "metarhia-jstp": "^1.1.0",
+    "iconv-lite": "^0.4.24",
+    "maojian": "^0.0.6",
+    "metarhia-common": "^0.0.38",
```

Changes in behaviour of `subst` in `metarhia-common@0.0.32` are incompatible with templating in `impress@0.5.x`. This particular change was introduced in https://github.com/metarhia/common/commit/d594dc988847e3fe3d6ada853d83d30a40c0f649.
```suggestion
    "metarhia-common": "0.0.31",
```